DigraphyApi/Properties/launchSettings.json

```
@@ -0,0 +1,28 @@
+{
+  "iisSettings": {
```

Just another thing I have in my other .NET project

```suggestion
{
  "$schema": "http://json.schemastore.org/launchsettings.json",
  "iisSettings": {
```